### PR TITLE
Bump BSI for core

### PR DESCRIPTION
**Pull request by AllanKerr**

### CODE REVIEW

**Comment by svanherk** [comment]
This is the second build today that I've seen timeout.  We might have to change it to `travis_wait 30 npm run build`, but that's not good 😞 .  We need the unbundled builds!

File: package-lock.json [hunk 1]
```diff
@@ -1008,9 +1008,9 @@
       }
     },
     "@brightspace-ui-labs/accordion": {
-      "version": "2.0.3",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/accordion/-/accordion-2.0.3.tgz",
-      "integrity": "sha512-qa0+Sol3rcfuCW6Htm3Lvotb8XIBUHoCwDBZDiIzzYsAvc+Iqg/TtcQEiluHSUA7o5Xw9G6HJrowCKAPBE3H3g==",
+      "version": "2.1.0",
```
Comment (AllanKerr) [inline package-lock.json:1011]: @cbelan-d2l

File: package-lock.json [hunk 2]
```diff
@@ -1058,9 +1058,9 @@
       }
     },
     "@brightspace-ui-labs/multi-select": {
-      "version": "3.1.0",
-      "resolved": "https://registry.npmjs.org/@brightspace-ui-labs/multi-select/-/multi-select-3.1.0.tgz",
-      "integrity": "sha512-y+6dZnNE8N/2uzY0Rx41l4h6MDyFKHgD/iZFnT9vbY8LpoA50gLEctY/N3rHZpVDAf4cUXiL4XeSwJ4Ub8LZCQ==",
+      "version": "3.1.1",
```
Comment (AllanKerr) [inline package-lock.json:1061]: @jstefaniuk-d2l

File: package-lock.json (hunk 1 above)
Comment (cbelan-d2l) [reply package-lock.json:1011]: Happy to see this is already in :)

**Pull request merged**